### ISSUE TITLE
with respect to how to choose an action

### ISSUE THREAD

**Issue by obitoquilt** [ikostrikov/pytorch-a3c#70]
"action = prob.multinomial(num_samples=1).detach()" in 59 lines of train.py. may i use epsilon-greedy  strategy to choose an action?
